src/actions/index.js

```diff
@@ -13,7 +15,35 @@ const fetchFormCategories = () => ({
   }
 });
 
-export const loadFormCategories = () => dispatch => dispatch(fetchFormCategories());
+export const loadFormCategories = () => (dispatch, getState) => {
+  const { formCategories } = getState().entities;
+  if (Object.keys(formCategories).length) {
+    return null;
+  }
+  dispatch(fetchFormCategories());
+};
+
+export const FORMS_REQUEST = 'FORMS_REQUEST';
+export const FORMS_SUCCESS = 'FORMS_SUCCESS';
+export const FORMS_FAILURE = 'FORMS_FAILURE';
+
+const fetchForms = (category, refetch) => ({
+  category,
+  [CALL_API]: {
+    types: [FORMS_REQUEST, FORMS_SUCCESS, FORMS_FAILURE],
+    endpoint: `form_categories/${category}/forms`,
+    schema: Schemas.FORM_ARRAY,
+    refetch
+  }
+});
+
+export const loadForms = (category, refetch) => (dispatch, getState) => {
+  const { forms } = getState().entities;
+  if (Object.keys(forms).length && !refetch) {
+    return null;
```

Comment:
Same here, returning null seems odd. Maybe we can restructure this if statement so we don't have to.

Reply:
Will do like the comment above